exporter/elasticsearchexporter/README.md

```diff
@@ -532,6 +532,8 @@ PUT _component_template/metrics-otel@custom
 }
 ```
 
+After installing this component template, if you have previously ingested data, you'll have to wait until the time series data stream's old index's `end_time`. It may take up to 30 minute by default, see [time series data stream look ahead time](https://www.elastic.co/guide/en/elasticsearch/reference/current/tsds-index-settings.html).
```

Comment:
```suggestion
After installing this component template, if you've previously ingested data, you'll need to wait until the old index of the time series data stream reaches its `end_time`. This can take up to 30 minutes by default. See [time series data stream look ahead time](https://www.elastic.co/guide/en/elasticsearch/reference/current/tsds-index-settings.html) for more information.
```

Does this sound better?